### PR TITLE
Bugfixes, documentation updates and Travis

### PR DESCRIPTION
**Pull request by owenjonesuob**
I decided to use [Travis CI](https://travis-ci.org/) to make sure everything about BANEScarparkinglite was absolutely great.

**Narrator:** *Things were not absolutely great.*

---

**Bugfixes:**
* `get_events_detail` now correctly references functions from other packages, and it has a shiny new progress bar
* `get_events` would fail if you tried to start from the first day of a month. It doesn't do that any more. And it was jealous of  `get_events_detail`'s progress bar, so it has one too.
* `get_rugby` just plain wasn't working. It is now.
* Examples in documentation have been corrected.

**Other things to note:**
* Updated to MIT license (same as `parking` repo in general)
* @rkenning has proper credit as an author/contributor
* There's a link to the development version (and a Travis build status badge) at the bottom of the README

### CODE REVIEW

File: r/BANEScarparkinglite/R/web_scraping.R [hunk 1]
```diff
@@ -352,11 +367,15 @@ get_events_detail <- function(from, to) {
...
 
+
 #' Scrape daily weather records for Bath
-#' 
+#'
 #' This function scrapes the \href{https://www.wunderground.com/}{Wunderground}
 #'  website to get daily weather summaries for Bath over a given date range.
-#' 
+#'
+#' @note The website can only display up to 398 records in one go; asking for
```
Comment (kinbiko) [inline r/BANEScarparkinglite/R/web_scraping.R:376]: Well that's an odd number...

**Review by kinbiko — CHANGES_REQUESTED** [review]
Nice! Travis CI is a great tool for open source projects. I looks like it points to the repo under your profile though, rather than anything set up for this repo. I don't really mind, but it's not super accurate - as:

1. changes that break in your repository doesn't break in this repo
1. breaking changes in this repository does not get reflected in the travis tag

So adding a label indicating that this is YOUR repo's build should suffice. And on that note, I don't see what would make a build fail/pass just by reading that `.travis.yml` file. Would you be able to document this (you can use `#comments` in yml files)? Also, could you link the button so that when you click on it you go to the travis build page? I think the link is: `https://travis-ci.org/owenjonesuob/BANEScarparking`

File: r/BANEScarparkinglite/R/web_scraping.R (hunk 1 above)
Comment (owenjonesuob) [reply r/BANEScarparkinglite/R/web_scraping.R:376]: Quite the contrary.
```
> 398 %% 2
[1] 0
```

**Comment by owenjonesuob** [comment]
**Changes made:**
* `.travis.yml` relocated to main directory. I've added some comments about what's going on in the file and a quick explanation of what Travis is doing/how to make sure builds pass.
* Separate badges in BANEScarparkinglite README for main and dev versions, each linked to respective Travis build pages

Were you asking for a Travis build for *this* version too? I sort of assumed so, and I've made changes accordingly. I'll need your help though - I don't have access to  BathML on Travis, so you'll have to go and flick the switch!

**Comment by kinbiko** [comment]
In the interest of avoiding over-engineering, let's not have a badge for the BathML build. Everything else looks good.

**Pull request merged**